packages/kit-bg/src/services/ServiceFirmwareUpdate/ServiceFirmwareUpdate.ts

```
@@ -785,16 +785,16 @@ class ServiceFirmwareUpdate extends ServiceBase {
           }
           const deviceExist = bootloaderMode
             ? // bootloader mode does not have connect id for classic
-              (response.payload ?? []).length > 0
+            (response.payload ?? []).length > 0
```

ESLint

[error] Insert `··`

(prettier/prettier)